src/app/views/template_edit.php

```diff
@@ -7,6 +7,7 @@ function get_field($name, $meta, $data) {
 
         switch ($meta['type']) {
             case 'color':
+                if (!$data) $data = 'rgb(0, 0, 0)';
```

Comment:
It's generally nicer to not have inline conditional statements. Could you please put the statement into a pair of braces?

```php
if (!$data)  {
    $data = 'rgb(0, 0, 0)';
}
```